src/components/CustomInput/CustomInput.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import styles from './CustomInput.module.scss';
+
+
+const CustomInput = ({ type, placeholder, changeHandler, label, id, value, required }) => (
+  <div className={styles.inputContainer}>
+    {
+      label ? (
+          <label className={styles.label} htmlFor={id}>{label}</label>
+        ) : null
+    }
+    <input
+      className={`${styles.input} ${label ? styles.withLabel : ''}`}
+      type={type}
+      placeholder={placeholder}
+      onChange={changeHandler}
+      name={label}
+      id={id}
```

Comment:
do you really need it ?
if you want to access this input somehow - better to use refs